README.md

```diff
@@ -1,30 +1,163 @@
 
-### `yarn start`
+## 📌 Prerequisites
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### 💻 1. System requirement :
+1. Any system with basic configuration.
+2. Operating System : Any (Windows / Linux / Mac).
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+### 💿 2. Software requirement :
+1. Updated browser
+2. Node js installed (If not download it [here](https://nodejs.org/en/download/)).
+3. Any text editor of your choice.
 
-### `yarn test`
+### ⚡ 3. Skill set :
+1. Knowledge of git & github.
+2. JavaScript
+3. ReactJS
+4. Mobile development Experience in Any Tech (optionally)
+5. Ionic / Capacitor (optionally)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 📌 Setting up Locale Environment
 
-### `yarn build`
+### 🚩 Forking repository :
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+1. Fistly you have to make your own copy of project. For that you have to fork the repository. You can find the fork button on the top-right side of the browser window. (Refer image below ) 
+2. Kindly wait till it gets forked.
+3. After that copy will look like `<your-user-name>/edu-client` forked from `codeforcause/edu-client`.
+<img src="./doc_imgaes/fork.png" alt="Demo fork image" align="center"/>
 
-The build is minified and the filenames include the hashes.\
+### 🚩 Clone repository :
+
+1. Now you have your own copy of project. Here you have to start your work.
+2. Go to desired location on your computer where you want to set-up the project.
+3. Right click there and click on `git bash`. A terminal window will pop up
+4. Type the command `git clone <your-fork-url>.git` and hit enter.(Refer the image to copy url)
+5. Wait for few seconds till the project gets copied
+<img src="./doc_imgaes/copy.png" alt="Demo copy image" align="center"/>
+
+### 🚩 Setting up remote :
+
+1. Now you have to set up remote repositories
+2. Type the command `git remote -v` this will list the remote connection to your repo.
+3. It will show something like this:
+```git
+
+origin  https://github.com/<your-user-name>/edu-client.git (fetch)
+origin  https://github.com/<your-user-name>/edu-client.git (push)
+
+```
+4. Now type the command `git remote add upstream https://github.com/codeforcauseorg/edu-client.git` this will set upstream as main directory
+5. Again type in command `git remote -v` to check if remote has been set up correctly
+6. It should show something like this :
+```git
+
+origin  https://github.com/<your-user-name>/edu-client.git (fetch)
+origin  https://github.com/<your-user-name>/edu-client.git (push)
+upstream        https://github.com/<your-user-name>/edu-client.git (fetch)
```

Comment:
Line 67 & 68 should be like this :
![image](https://user-images.githubusercontent.com/55068936/110891459-0ddd6900-8318-11eb-80a4-9c6f7fb02f83.png)

Reply:
what should I do to make it look like that?

Reply:
oh yes, done